backend/app/routes/schemas/bot.py

```diff
@@ -39,6 +51,8 @@ class BotInput(BaseSchema):
     instruction: str
     description: str | None
     embedding_params: EmbeddingParams | None
+    generation_params: GenerationParams
```

Comment:
Regarding the policy, could you make it nullable? (Sorry for making you confused)
Where our policy is:
- Output Schema should be NOT nullable
- Input Schema should be nullable (It makes stronger dependency with frontend because frontend side need to prepare default values)